keyboards/clawsome/bookerboard/readme.md

```diff
@@ -0,0 +1,3 @@
+/* Copyright 2020 AAClawson (AlisGraveNil)
```

Comment:
This file should match the [template](https://github.com/qmk/qmk_firmware/blob/master/quantum/template/avr/readme.md).

Reply:
I think I got it right